plugins/gcpaudit/walkthrough.md

```diff
@@ -0,0 +1,389 @@
+
+As the name suggests, the GCP Audit Logs plugin ingests GCP Audit Logs for several key GCP services. 
+
+The  plugin uses an optimized GCP logging sink to send the most critical events from the monitored services to a user-defined Pub/Sub subscription. The GCP Audit Logs plugin subscribes to the Pub/Sub topic. It forwards the events from Pub/Sub to the Falco engine, which then filters the events and enriches the output with [custom metadata](https://github.com/falcosecurity/plugins/tree/master/plugins/gcpaudit#supported-fields). 
+
+<img style='border:1px solid #000000' src="./images/architecture.png" alt="A diagram showing how the plugin works with other services"/>
+
+The filtering process is based on a set of custom Falco rules authored with the [Mitre Att&ck framework](https://www.mitre.org/focus-areas/cybersecurity/mitre-attack) in mind. When an action, such as deleting a VM, triggers one of the rules, Falco sends out an alert. 
+
+### GCP Audit Logs walkthrough
+
+In this section, you will learn how to install and use the GCP Audit Logs plugin. You’ll first configure the necessary GCP services. This includes setting up a Pub/Sub topic and subscription and creating a GCP Logging service sink. Then, you will build the plugin. Next, you’ll install Falco and configure it to use the plugin. Finally, you’ll test everything to ensure it’s all working correctly. 
+
+#### Prerequisites
+
+To follow along with this tutorial, you will need a GCP project and an account with appropriate permissions, including the ability to:
+
+
+
+* Create and manage compute instances
+* Create and manage pub/sub topics
+* Access GCP audit logs and create custom GCP Logging sinks
+
+You will also need to have the gcloud command line tool installed on your local machine or use Cloud Shell (which already has gcloud installed). 
+
+Make sure you know the project ID of your GCP project. 
+
+Finally, to build the plugin, you will need to have both Git and Go 1.17 or greater. If you use Cloud Shell, both tools are already installed for you. 
+
+
+#### Step 1: Configure the GCP services
+
+In this section, you’ll configure the necessary backend GCP services. 
+
+**NOTE**: This walkthrough creates resources that could incur charges against your GCP bill. The easiest way to reduce charges is to use a project that can be deleted when you are finished. Alternatively, you will need to delete the resources you created manually upon creation. There are instructions on how to do this at the end of the walkthrough. 
+
+To begin make sure you’re authenticated to your GCP account in gcloud and you’ve selected the appropriate project.
+
+
+```
+    gcloud auth login 
+    gcloud config set project [project id]
+```
+
+
+It is helpful later on to have your GCP project ID configured in an environment variable.
+
+	`PROJECT_ID="[project id]"`
+
+The next step is to create a Pub/Sub topic to receive the output from the GCP Audit logs. 
+
+	`gcloud pubsub topics create falco-plugin-topic`
```

Comment:
surround the code blocks with ``` or it will create strange patterns:

![image](https://github.com/falcosecurity/plugins/assets/4520100/e039b56d-1db2-4fbf-b57c-faf35fc21672)